.github/workflows/release.yml

```diff
@@ -39,14 +39,14 @@ jobs:
           sh update_version.sh
```

Comment:
Nitpick: please change the name of the workflow, so in the first line the name should be:
```
name: Create core module release
```